Fantome.League/IO/MapParticles/MapParticlesFile.cs

```diff
@@ -7,24 +7,29 @@ public class MapParticlesFile
     {
         public List<MapParticlesParticle> Particles { get; private set; } = new List<MapParticlesParticle>();
```

Comment:
Would be better to initialize it only in 'MapParticlesFile(string fileLocation)' constructor ;)